### PR TITLE
fear: add vlt to members.csv

### PR DESCRIPTION
**Pull request by darcyclarke**
> Note: Uploading several assets just in case

logo-horizontal-black
logo-horizontal
vlt-logo-white
vlt-logo-black
vlt-logo-square-background

### CODE REVIEW

**Review by vladh — APPROVED** [review]
Lovely to have you! :) LGTM, waiting for second review.

**Comment by chadwhitacre** [comment]
Looks pretty good!

> vlt /vōlt/ is new home for Open Source JavaScript Software.

This reads like a typo to me, should be "_the_ new home"?

@vladh Darcy and I have been working on this for a while, we should bring them in as Innovators.

**Pull request merged**
